### PR TITLE
Update manage-event-based-updates-windows-defender-antivirus.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2472
Added note to Microsoft Maps

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-antivirus/manage-event-based-updates-windows-defender-antivirus.md [hunk 1]
```diff
@@ -143,6 +143,9 @@ If you have enabled cloud-delivered protection, Windows Defender AV will send fi
 4. Expand the tree to **Windows components > Windows Defender Antivirus > Signature Updates** and configure the following:
     1. Double-click **Allow real-time definition updates based on reports to Microsoft MAPS** and set the option to **Enabled**. Click **OK**.
     2. Double-click **Allow notifications to disable definitions based reports to Microsoft MAPS** and set the option to **Enabled**. Click **OK**.
+    
+> [!NOTE]  
+> By allowing Microsoft to disable local Defender virus definitions, it has identified as causing false-positive detections, through the client's cloud connection to MAPS. You must configure your computer to join Microsoft MAPS for this functionality to work.
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-antivirus/manage-event-based-updates-windows-defender-antivirus.md]:
```suggestion
> By allowing Microsoft to disable local Defender virus definitions, it has been identified as causing false-positive detections through the client's cloud connection to MAPS. You must configure your computer to join Microsoft MAPS for this functionality to work.
```
Maybe I have misunderstood the phrasing here, but this suggestion is based on how I interpret the sentence context.

**Comment by illfated** [comment]
Thank you for adding my suggestion. I admit that I struggled to understand the context, in particular the meaning of "By allowing Microsoft to disable local Defender virus definitions" and its linked meaning with the rest of the sentence. Then again, I presume it is easier to interpret and see the context for people who already are immersed in this topic by working with this software package on a daily or weekly basis.

**Comment by e0i** [comment]
@Justinha Kindly review and merge this PR that has passed copy review. Thanks.

**Comment by Justinha** [comment]
@thanks let me get input from a better expert.

File: windows/security/threat-protection/windows-defender-antivirus/manage-event-based-updates-windows-defender-antivirus.md [hunk 2]
```diff
@@ -143,6 +143,9 @@ If you have enabled cloud-delivered protection, Windows Defender AV will send fi
 4. Expand the tree to **Windows components > Windows Defender Antivirus > Signature Updates** and configure the following:
     1. Double-click **Allow real-time definition updates based on reports to Microsoft MAPS** and set the option to **Enabled**. Click **OK**.
     2. Double-click **Allow notifications to disable definitions based reports to Microsoft MAPS** and set the option to **Enabled**. Click **OK**.
+    
+> [!NOTE]  
+> By allowing Microsoft to disable local Defender virus definitions, it has been identified as causing false-positive detections through the client's cloud connection to MAPS. You must configure your computer to join Microsoft MAPS for this functionality to work.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-antivirus/manage-event-based-updates-windows-defender-antivirus.md]:
```suggestion
> "Allow notifications to disable definitions based reports" enables Microsoft MAPS to disable those definitions known to cause false-positive reports.  You must configure your computer to join Microsoft MAPS for this function to work.
```

**Comment by illfated** [comment]
Thank you for the recent text update. It makes a lot more sense to read it now. 👍

**Pull request merged**